<!DOCTYPE html>
<html lang="en" style="scroll-behavior: smooth;">
<head>
    <meta charset="UTF-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>My_linkpage</title>
</head>
<body>
    <h1>Welcome to About us page</h1>
    <a href="day2.html">Go to about us page</a><br><br>
    <a href="https:facebook.com" target="_blank">Go to facebook</a>
    <br><br>
    <a href="www.linkedin.com" target="_blank">Go to linkedin</a>
    <br><br>
    <a href="tel:+2348101806661">call me</a>

    
    
    <p>CHAPTER ONE

       <h1 id="Top"> INTRODUCTION</h1> 
       <a href="#bottom">Go to Bottom</a>
       
       <P> Bacteria a iful microbial communities in the soil, where they play fundamental roles in the maintenance of the soil ecosystem, recycling soil nutrients, minerals enrichment, nitrogen fixation, and improving the soil biogeochemical properties, which is necessary for crop production. [Delgado-Baquerizo, et al., 2018].
        The bacteria are the most abundant group usually more numerous than the four combined. Soil bacteria can be rod, (bacilli) cocci (spherical) spirilla (spirala), of
        These, bacillus are more numerous than the other. They are one of the major groups of soil bacteria population and are widely distributed. [Bhagabati A, et al., 2004].
        The number and type of bacteria present in a particular soil would be greatly influenced by geographical location such as soil temperature, soil type, soil ph. organic matter contents, cultivation, aeration and moisture content. [Davies C, et al., 1999].
        Member of the bacillus genus are generally found in soil and most of these bacterial have the ability to disintegrate protein, namely proteolytic activity. Proteases of these microorganism play an important role in the nitrogen cycles, which contributes to the fertility of the soil. In paddy field soil most of the nitrogen source is stored as biomass protein and decomposes slowly to low molecular weight amino acids by the activity of soil protease.
        Soil protease is thought to be mainly supplied by soil microorganism there are numbers of bacteria having potential to produce antibiotic, example of which is bacillus species which produce antibiotic like bacitracin, pumolin and gramicidin which are active gram positive bacteria such as staphylococci, streptococci, coryne bacter, steptomyces specie which produce antibiotic like tetracycline, chloramphenicol, vancomycin, gentamycin which are active against gram negative bacteria.
        Antibiotic resistance is a specific type of drug resistance virtually evolves via natural selection acting upon random mutation, but it can also be engineered by applying an evolutionary stress on a population.
        The use of antibiotic along with immunization and sterile technique in hospital has significantly decreased the number of lethal bacterial infection.
        However, antibiotic once seen as miracle drugs are now becoming useless in treating various bacterial diseases. The rapid emergence of resistance to antibiotic among pathogen generate visions of the potential post antibiotic era threating present and future medical advance [Wise R, et al., 2008].
        The emergence of bacteria resistant to antibiotics is common in areas where antibiotics are used, but antibiotic resistant bacteria are also increasingly occurs in aquatic environment. The widespread use of antibiotic in medicine and in intensive animal husbandry is indicative of selection pressure exerted on bacteria. Intensive animal husbandry causes resistant bacteria to enter the environment directly from liquid manual and muck.
        Antibiotic are one of the secondary metabolites produced by microorganism and represent the first choice usually selected to manage microbial infections of bacteria origin.
        In the same respect, most of the widely used antibiotic are produce from a limited genera of microorganism, mainly penicillium sp., Streptomyces sp., cephalosporium sp., and bacillus sp., 
        These antibiotic-producing microorganism are widely studied to identify new forms of antimicrobial compound as alternative to combat emerging multidrug-resistant microorganism.
        Technically defined, antibiotics include a chemically heterogeneous group of small organic molecules of microbial origin that, at low concentrations, are deleterious to the growth or metabolic activities of other microorganisms. Over the past decade, however, genetic and molecular techniques, have been applied to demonstrate conclusively that microorganisms synthesize a variety of antibiotics, even under field conditions, in the rhizosphere i.e. that portion of the soil enriched in carbon and energy resources released by plant roots.  That Soil is rich in microorganisms capable of antibiotic synthesis is well accepted, but the frequency with which synthesis occurs at ecologically significant levels in nature has been much less clear. [Brun, et al., 2000]. 
        Actinomycetes are best known for their ability to produce antibiotics and are gram positive bacteria which comprise a group of branching unicellular microorganisms. Among actinomycetes, the Streptomycetes are the dominant. Important species of bacteria known to produce antibiotics is the Bacillus spp. Over 5000 antibiotics have been identified from the culture of  gram positive, gram negative organisms and  filamentous fungi, but only a few have been  commercially used to treat human, animal and plant Diseases. The genus Streptomycete is responsible for the formation of more than 60% of known antibiotics [Williams, et al., 1993].
        The trend of search for antibiotics in the past and in recent times as a result of drug resistance by microbial species has required combing the earth for various sources of antibiotics including the soil.
        In addition many antibiotic-producing microorganism could also be isolated from soil, Antibiotics are one of the secondary metabolites produced by microorganisms and represent the first choice usually selected to manage microbial infections of bacterial origin. In the same respect, most of the widely used antibiotics are produced from a limited genera of microorganisms, mainly Penicillium sp., Streptomyces sp., Cephalosporium sp., and Bacillus sp. [Zinsser, et al., 1992].
        These antibiotic-producing microorganisms are widely studied to identify new forms of antimicrobial compounds as alternatives to combat emerging multidrug-resistant microorganisms. Soil bacteria represent the primary source of polypeptide antibiotics that are used against various pathogens, particularly those that are drug resistant. [Amin, et al., 2012]. Bacillus is the major bacterial genus that exists in soil, and it produces many valuable antibiotics [Tabbene, et al., 2009].
        Antibiotics extracted from Bacillus sp. have been tested to define their antimicrobial spectrum. Radical differences exist among the antimicrobial compounds produced by Bacillus, as some exhibit a narrow spectrum of activity, whereas others have a broad spectrum [Caulier, et al., 2019; Hover, et al., 2018].
        Bacillus subtilis is a major bacterial species that produces effective antimicrobial compounds against multidrug-resistant pathogens such as methicillin-resistant Staphylococcus aureus (MRSA) [Chalasani, et al., 2015].
        MRSA is the main pathogen that causes nosocomial infections resulting in bacteremia associated with high morbidity and mortality rates [Valaperta, et al., 2010]. The danger of MRSA is not only because of its being a frequent cause of hospital infections but also because of its increasing ability to acquire resistance to various antibiotics, including ß-lactam, aminogly cosides, fluoroquinolones, clindamycin, chloramphenicol, and tetracycline [Aqil, et al., 2001].
        [Hashem, et al., 2013; Guignard, et al., 2005; and Gardete, et al., 2014; Siberry, et al., 2003].
        Thus, there is an urgent need to discover and develop new antibiotics because of the increasing frequency of MRSA resistance to vancomycin and teicoplanin, which are the final therapeutic options against this pathogen [Hsieh, et al., Medicine 2016; and Tarai, et al., 2013].
         
        In the present study, we identified an antibiotic-producing bacterium from a screen of bacterial species isolated from soil samples by sequencing the 16S ribosomal DNA gene. The isolated bacterium was found to belong to the genus Bacillus. We further investigated the optimal growth conditions for antibiotic production, stability in the presence of various metals, and its growth over a wide range of temperatures and pH levels. Furthermore, we examined the chemical identity of the antibiotic and evaluated its antimicrobial activity in vitro and in vivo.
        
        
        
        1.2 SCOPE OF THE STUDY
        This dissertation work base on the isolation of bacillus spp, from Olabisi Onabanjo University, Ago Iwoye farm soil and their Screening for antibiotic producing abilities.
        1.3 AIM AND OBJECTIVE OF THE STUDY
        •	To identify antibiotic producing bacteria from a screen of bacteria species isolate from soil sample.
        •	To examine the biochemical test identity of bacteria and evaluate its antimicrobial activity.
        •	To investigate the optimal growth conditions for antibiotic production, stability in the presence of various metals and its growth over a wide range of temperature. 
        CHAPTER TWO
        2.0	 LITERATURE REVIEW
        Antibiotics are a natural substance of biological, synthetic or semi-synthetic origin [Barrow et al., 2003]. In 1928, the term antibiotic appeared as antibiosis in French microbiological literature whereas as later in 1942 the term antibiotic was introduced by Waksman.
        The demand for new antibiotics growing day by day due to the emergence of multiple pathogens that are resistant to antibiotics cures for formerly life-threatening diseases. In recent years several microorganisms that are able to produce antibiotics are grown on the artificial media for the intensive search for antibody producing microorganisms. 
        The most important antibiotics include Amino-glycosides, Penicillin, Macrolides, Glycopeptides,Cephalosporins and Tetracyclines (Medicinal research reviews 2009.)
        Soil is a complex and very diverse environment providing versatile source of antibiotic producing organisms [Baltz, 2006]. Each year nearly 500 antibiotics were found, in which 60% of antibiotics are obtained from the soil [G: Natural products in drug discovery, present status and perspectives. Pharmaceutical Biotechnology, 2009]. Recent analyses have shown that screening of soil for antimicrobial activities have been carried out in many parts of the world [Makut et al., 2011]. A tea-spoon of soil contains hundred million to one billion bacteria active in each acre of the soil. Bacillus species are gram positive, rod shaped, sporulating and aerobic or facultative anaerobic bacteria that were most abundant bacterial strains found in the soil [Hussein et al., 2006]. Which were capable of producing dozens of antibiotics [Waites et al., 2008]. Genus of Bacillus was interesting to investigate and were considered microbial factory for the production of biologically active secondary metabolites [Lisboa et al., 2006]. Various studies confirmed Bacillus species to produce antimicrobial compounds having pharmaceutical and biotechnological importance [Awais et al., 2007]. The  main antibiotic  producer  of  Bacillus sp. were B. cereus (Zwittermicin, Cerexin) B. brevis (Tyrothricin, Gramicidin), B. circulans (Circulin), B. licheniformis (Bacitracin) B. laterosporus (Laterosporin), B. polymyxa (Colistin, Polymyxin), B. subtilis (Bacitracin, Polymyxin, Subtilin, Difficidin, Mycobacillin,) B. pumilus (Pumulin) are mainly polypeptides Reddy PP: Plant growth promoting rhizobacteria for horticultural crop protection. Springer India 2014 were mostly active against gram positive bacteria. An antibiotic activity of B. pumilus and B. subtilis has been reported against Staphylococcus aureus and Micrococcus luteus [Tabbene et al., 2009]. Bacillus species isolated from farm soils shows antibacterial activity against the methicillin-resistant Staphylococcus aureus. Cellulases, Subtilisins, and Amylases produced by B. subtilis had several industrial importances and were consumed by the laundry industries. Fungi were extremely diversified worldwide. On earth, there are approximately 1.5 million species of the fungi out of which 70 - 100,000 species were discovered and 95% of these have yet to be discovered. Fungi are best source of antibiotics and to search antibiotics from them is very promising. Twenty best-selling medical drugs worldwide were fungal derived Concepcion GP, [Lazaro and Hyde, et al., 2001].
        According to the International center of information on antibiotics, 338 species of fungi are able to produce antibiotics. In soil habitat, fungi compete against different other microorganisms and may induce antimicrobial production. Filamentous fungi contaminated with broad spectrum antibiotics, inhibited by antibiotic resistant bacteria are able to produce the secondary metabolites. 20% of isolated antibiotics have been discovered from soil fungi. Many species of fungi including Penicillium, Aspergillus, Cladosporium and Yeasts which were reported only to be the food Cantrell SA and Baez-Félix C: Fungal molecular diversity of a Puerto Rican subtropical hypersaline microbial mat. Fungal Ecology 2010; 3(4): 402-405.are tremendous sources of industrially important enzymes and secondary metabolite production. Antibiotics produced by the fungal species are widely used inthe chemotherapy especially Fusidic acid, Cephalosporin and Penicillin which have both antifungal and antibacterial activity. Fungi represent an important source of potentially powerful new pharmaceutical products Wasser SP: Medicinal mushrooms as a source of antitumor and immunomodulating polysaccharides. [Applied microbiology and biotechnology 2002]. Nowadays, Pharmaceutical industries are investigating an array of fungi to increase the number of discoveries. For many years ago, in many societies fungi have been recognized as palatable nutritious food and are now acceptable as a valuable source in pharmaceuticals for development of the medicines. Several thousand antibiotics have been isolated from soil microorganisms since the discovery of penicillin, but unfortunately these have been limited only to fifty, most of them being too toxic to humans [Carlile et al., 2000].
        According to the food and drug administration approximately 80% of antibiotics produced from natural habitat are fed to animals and only 20% of them are used to treat infections in the humans. The objective of this study was to find antibiotic producing microorganisms and to check their ability to inhibit the growth of S. aureus and E. coli. Bacillus bacterial belong to the firmicutes phylum, the Bacillaceae family, and are Gram-positive bacteria that can be found in diverse habitats, but mostly in farm soil [slepecky and Hemphill, 2006; Graumann 2012]. Their cells are straight, rod shaped and can be either found in pair or singles, chain or even as long filaments as well as peritrichous flagella in some motile species [Baruzzi et al., 2011].
        
        Moreover, for each of the cells, an endospore form to resist adverse and harsh conditions such a radiation, heat, and chemicals such as disinfectants (tan and Ramamurthi, 2014). Although most species are facultative aerobes, some can be anaerobic [Prieto et al., 2014]. Several studies have revealed that some species such as Bacillus brevis, Bacillus subtilis, Bacillus circulans, Bacillus polymyxa, Bacillus cereus, and Bacillus licheniformis are capable of producing antibiotics (Hirad et al., 2013; Sawale et al., 2013; Rai et al., 2017). Some of them were extracted and used for many medical treatments, for example Tyrothricidin for sore throat infections and Polymyxin for ear inflammations [Mondol et al., 2013].
        Antibiotics are produced during the early stages of spore formation, which suggest that this phenomenon is considered as one defense mechanism of these bacterial species against other microorganisms [Sumi et al., 2015]. In addition, the antibiotics synthesized by Bacillus species can be classified into four categories according to their mechanism of action: cyclic oligopeptides (for example,bacitracin) that hinder cell wall synthesis; linear oligopeptides (for example, gramicidins and polymyxin) that interfere with cell membrane function; basic peptides (for example, edeines) that restrain the initiation complex formation on the small ribosome subunit; and aminoglycoside antibiotics that distress ribosome function [Graumann, 2012]. Various scientific literature have revealed that most Bacillus species isolated from marine sediments showed more antagonistic activities against Gram-positive bacteria such as Staphylococcus aureus and Staphylococcus epidermidis, but however less toward Gram-negative bacteria such as Pseudomonas aeruginosa [Kannahi and Eshwari, 2016].
        This phenomenon is supported by recent studies in India and the Arabian Gulf [Parvathi et al., 2009; Hirad et al., 2013; Sawale et al., 2013]. In addition, this case could be also explained due to differences in cell wall composition of tested bacteria. Unlike Gram-positive bacteria which has only one thick layer of peptidoglycan, Gram-negative bacteria has far more complex components, in which the antimicrobial peptides must cross to interfere with its molecules required during cell wall biosynthesis ([Malanovic and Lohner, 2016].
        Moreover, lantibiotics inhibit growth of the Grampositive bacterium S. aureus via the obstruction of a carrier of peptidoglycan monomers, Lipid II, across the cytoplasm membrane to the exterior side. As a consequence, the incorporation of the growing peptidoglycan network by Lipid II is prevented during cell wall synthesis [Baruzzi et al., 2011; Sumi et al., 2015]. Another possible explanation for the antimicrobial peptides‟ action is that these disrupt the cell cytoplasm by binding to and aggregating with the membrane. As a result, these form pores through which ions can pass and lead to cell components leakage and eventually, cell death (Sumi et al., 2015).
        The main aim of this study was to isolate, for the first time, Bacillus species from OOU farm soils that exhibit antimicrobial activities against both Gram-positive and Gram-negative bacteria, as well as to investigate their production in liquid media.
        
        2.1	 Screening of Antibiotic Producing Bacterial
        Antibiotics can be defined as any chemical substance of small organic molecules which at low amount are harmful to the metabolic activities and growth of other microbes [Bron and skimkets, 2000]. The natural products obtained from microorganism are still appearing as the most auspicious source of the future antibiotics [palaez, 2006]. Antibiotics which are mostly in use today are the nature derivatives of fungi and Actinomycetes [Tawiah et al., 2012].
        In nature, there is universal dissemination of antibiotics among the microorganisms responsible for their antagonism [Evanorasetr et al., 2010]. There are numbers of bacterial which are capable of producing antibiotics which include Bacillus [Waites et al., 2008]. Actinomycetes [Abdulkadri and Waliyu, 2012; Tiwari and Gupta, 2013] pseudomonas [Cartwright et al., 1995]. And Streptomyces [willey et al., 2008]. And most of these isolates are soil bacterial.
        Extensive use of antibiotics both for clinical and veterinary purpose leads to the development of resistance in many infectious strains [korboke et al., 1992]. With the continuation of this process of the process resistance, the search for new anti-infective drug should be carried out [thukur et al., 2017; brown and wright 2005]. In addition to the resistance problem, a number of new infection diseases have been discovered over the past 30 years [Zimmerman and Zimmerman, 2003; Bryskier, 2005]
        
        Soil is rich in microorganism which are capable of producing antibiotics [Bron and Skimets 2020].
        The traditional approach is random screening in which bacterial are isolated grow and their activity done for more than 50years still we are getting result in favour to us and thus we are sticking with this approach [wawrik et al., 2007].
        2.2 	Screening of soil isolates for antibiotic activity 
        A sensitivity assay was carried out to determine the antibiotic producing ability of some of the isolates against the other test organism isolated. The technique was aimed at demonstrate the alexander Flemings observation of the discovery of penicillin. For the bacteria sensitivity test, the isolate being screen for antibiotic production was streaked horizontally at a particular portion on sterile nutrient agar while the other organism on which the activity was tested against were streaked vertically very close to the screened one. Plate were incubated for 24hour at 370c, during which any substance produced by the tested organism growing perpendicularly and produce zone of inhibition.  
        
         
        
        
        
        
        
        
        
        
        
        
        
        
        
        CHAPTER THREE
        3.0	MATERIALS AND METHODS
        3.1 Materials 
        The materials used for this dissertation includes the following measuring cylinder, weighing balance, beakers, conical flasks, autoclave, incubator, aluminum foil, sterile petri dishes, inoculating loop and needles, microscope, macCartney bottle, test tubes, Durham tubes, cotton wool, spirit lamp, marker, ethanol, reagents distilled water, slide, cover slips, pasteur’s pipette, lighter, polythene bags, standard starter culture.  
        3.2	Collection of Samples
        A total number of 15 farm soil samples weighing approximately 10gram each were collected from different area of Olabisi Onabanjo University,Ago-iwoye Ogun State farm site and farm soil sample outside the school environment. All the sample were collected 4 to 5cm deep from the surface using spatula and placed in sterilized zip lock polythene bags. The samples were immediately transferred to microbiology laboratory, Olabisi Onabanjo University and kept at 4oc for further processing. 
        3.3	Microbial Analysis 
        3.3.1	Sterilization of Materials 
            An autoclave was used in carrying out all the various processes of sterilization ranging from glass apparatus to the media used. The sterilization was done at 121oc for 15minutes. Wire loop was sterilized by flaming to red hot over flame from spirit lamp. The surface of the working bench were sterilized by swabbing it with cotton wool soaked in 70% ethanol or methylated spirit.
        3.2.2	Preparation of Media 
        Two different media were used for the isolation of each sample, these are;
        Nutrient Agar (NA) medium
        MacConkey Agar Medium
        For MacConkey Agar Prepartion
        The MacConkey Agar media was prepared according to the manufacture’s instruction in which 28g of the agar was weighed using mechanical weighing balance and dissolved in 1000ml of distilled water into a labelled conical with non- absorbent  cotton wool and aluminum foil; it was then sterilized in an autoclave at 121oc for 15minutes. After sterilization it was then brought out and allowed to cool to about 45oc before pour plating.
        The same procedure was followed for the preparation of nutrient agar in which 28g was weighed using mechanical weighing balance and dissolved in 1000ml of distilled water into a labelled conical flask. The mouth or trip of the flask was corked with non-absorbent cotton wool and aluminum foil; it was then sterilized in an autoclave at 121oc for 15minutes. After sterilization it was then brought out and allowed to cool to about 45oc before pour plating.
        3.3.3		ISOLATION OF MICROORGANISM 
            Serial dilution 
        1 gram of farm soil sample was weighed aseptically using a mechanical weighing balance and then dissolved it in 10ml of sterile distilled water to make soil Suspension in test tube. The test tube was vortexes for 2 to 3minutes to remove soil, stones and debris. A serial dilution of 6 MarCartney bottle was made each containing 9ml of distilled water. The Marcartney bottle were labelled from 10-1 to 10-6 and were sterilized in an autoclave at 121oc for 15minutes. After sterilization, the bottles were allow to cool after which, the 1ml of the supernatant sample was aseptically added into the first bottle labelled 10-1 and was mixed thoroughly. 1ml of the mixture in the first 10-1 bottle was withdrawn using sterile needle and syringe and transferred into the second bottle 10-2 and the same procedure was repeated for the third bottle until it reaches the last bottle 10-6. After the serial dilution, 1ml of the mixture was withdrawn using sterile needle and syringe from 10-2 and 10-4 MacCartney bottles before pour plating into already sterile labelled petri dished. Each of the two different agar was poured into petri dished containing the same dilution of inoculum. All the petri dished were swirled carefully for even distribution of the organism and allowed to set. The plate were then allowed to solidified and placed in an inverted position to avoid condensation of water into the media and were incubated in an incubator at 37oc for 24hours for both nutrient agar and MacConkey agar.  
        After 24 hours of incubation period, all the plates were examined for growth and the number of colonies were counted and recorded using colony counter.
        Isolation were sub-culture by streaking method to obtain a pure culture. The pure culture were stored on fresh agar slants and incubated at room temperature or stored in a refrigerator at low temperature as stock culture for further tests.
        3.3.4	 Total viable Count 
        This was done immediately after inoculation period for 24-48 hours and all colors were counted and recorded. 
        3.3.5	Pour culturing 
        Isolates from the petri dished of both nutrient agar and MacConkey agar were further sub-cultured by streaking to obtain pure cultures. Pure cultures showed distinct colonies and not clumsy and colonies uniquely same in appearance and morphology.
        3.3.6	Enumeration, characterization and identification of isolates
        Enumeration, characterization and identification of isolates after incubation of the plates, each petri dished was observed. The counting of the colonies which represent the cells of specific microorganism was carried out and recorded. Characterization of the isolation was carried out by employing microscopic, physiological and biochemical tests.
        3.4. Micro morphological and Biochemical test
        Important biochemical and morphological tests and observations [Collins et al., 1989] were carried out as indicated in the result. Some of the tests carried out included Gram staining, oxidase test, catalase test, indole test, sugar fermentation test. The tests were carried out as a confirmation of the microbial identity of the test microorganism.
        3.4.1 BIOCHEMICAL TESTS
        3.4.2 Gram Staining
        The pure culture was stain and viewed, a thin smear of each isolate from 24 hours culture was made on a clear glass slide using a wire loop and fired by passing it through a flame thrice (3 times). Al the slide were labelled and was flooded with two drop of crystal violet solution and allowed to stay for 1 minute, excess stain was rinsed off with water and several drops, gram iodine are applied for 60seconds and rinsed off. The smear was flooded with acetone until no mote violet stain coloration was observed. This slide was washed with tap water and counter stained with 3 drop of safranin reagent and stay for 60 seconds, the stain was washed off with tap water and allowed to air dried for some minutes. A drop of immersion oil was applied and the smear was mounted on the microscope and examined using X100 oil immersion object of a light microscope.
        Gram position bacterial were characterize by purple coloration wile gram negative bacterial were pinkish in color. Observation with the microscope also determine the shaped of the bacteria whether shorter or longer rod or cocci and their arrangement.
        3.4.3 Catalase Tests 
        The slide were swabbed with ethanol and cotton and famed by passing it through a spirit lamp. A thin smear of each isolate from 24hours culture was made on the clean sterile glass slide with distilled water using an inoculating wire loop. A few drop of freshly prepared 3% hydrogen peroxide were added on the smear of different test organism. Evolution of gas while films caused by the liberation of free oxygen indicate catalase positive while the absence indicate negative.
        2H20 ----------------- 2H20 + 02
        3.4.4 Oxidase Test
        1g of oxidase powder was dissolved in 100ml of water. Using a freshly sub cultured organism from slants, a smear of each organism was made on a filter paper and a drop of the oxidase solution was put on it. Observation was made to check for color changes on any of the organism, the formation of a very deep purple within 10seconds indicates a positive reaction, while the absence of deep purple coloration show a negative reaction.
        
        3.4.5 Indole Test
        This test demonstrate the ability of certain bacteria to decompose the ammonia acid tryptophan to indole which accumulate in the medium. Bacteriological peptone water was prepared according to the manufacture’s instruction, and sterilized at 121oc in an autoclave for 15minutes. After sterilization it was allowed to cool and then 9ml was dispensed into already sterile MacCartney bottle and the isolates were grow in each bottle and then incubated for 24hours at 37oc after 24hours of incubated, 5-6 drops of Kovac’s reagent was added to the organism growth in the MacCartney bottles.
        Observation was made in which a reddish or rose pink color development at the top layer indicate a positive results of indole production while no color changes or gold coloration indicates a negative results.
        3.4.6 Sugar Fermentation 
        This test conducted to determine the utilization of different sugars since bacteria has its own ability to ferment sugar accompanied with the production of acids and gas. The sugar used were Glucose, Fructose, Lactose and Sucrose.
        Peptone water and the sugar were prepared according to the manufacture’s specification in which 15g of the peptone was dissolved in 1000ml of distilled water and 10g of a particular sugar was also dissolved into the same 1000ml of distilled water. Two mixtures was dissolved and mixed thoroughly and 3 drops of methyl red indicate was put on the solution for color changes to red color.
        5ml was withdrawn into the test tubes and all the tubes used were labelled and Durham tubes were inserted in an inverted position for gas production and all the labelled test-tube used were tightly corked with cotton wool. The test tube were then sterilized in an autoclave for 15minutes at 1210c. After 15minutes of sterilization, it was brought out of the autoclave and allowed to cool. The organism were picked each from the 24hour culture plate and were inoculated into all the labelled test tube after cooling and the incubated for 24-48 hour for color change. All the tube were observe after 24-48 hour of incubation for color changes from red to yellow which indicates positive reaction while no color changes remained red. The control tests for each of the sugar were also set up in test tube and no color changes was observed.  
        3.4.7 Isolation and Screening of bacteria
        Selected colonies based on morphology and zone of inhibition were plated directly unto nutrient agar for isolation of bacteria. After an overnight incubation discrete colonies of bacteria were selected based on colony character and streaked on new agar plates according to previous study [Iqbal et al., 2015]. The plates were then kept at 370c for 24hours.
        Multiple streak plate method is used previously [Iqbal et al., 2015]. Was applied to obtain purified cultures of bacteria on nutrient agar plates. Pure single colonies were noticed after 24hours incubation at 370c purified cultures of bacteria were identified based on colony morphology, microscopic characters and biochemical characters [Iqbal et al., 2015; yunus et al., 2016]. Following Bergey’s manual of determination bacteriology [Bergeys, 1984].
        3.4.8 Antibiotic susceptibility testing of target bacteria by disc diffusion method
        The organism cultured were subjected to antibiotic susceptibility test using the disc diffusion method as originally described by [Baueet et al., 1996]. All bacteria were culture on Tryptic Soy Broth (TSB) medium and incubated at the appropriate temperature for 24hour. Nutrient agar medium (10ml) was poured into each sterile petri dish (100mm diameter) and allowed to solidify. Four sterile swab was used to pick the cultured organism (Bacillus subtilis, Bacillus licheniformis, Streptomyces spp and Actinomycetes spp) and each organism was spread uniformly on four petri dish respectively. Multiple disk (Whatman no 1 filter) was soaked with Tryptic Soy Broth containing the “Target bacteria” and was placed on the petri dish aseptically with the aid of sterile forceps and the procedure was repeated for remaining Bacillus isolate. The petri dish were then incubated at 370c for 24hours in an inverted position. Diameter of zone of inhibition around each antibiotic disc were measured and recorded in millimeters.      
        CHAPTER FOUR
        4.0 RESULT
        The total Bacillus population counts in different farm soil sample from Olabisi Onabanjo University, Ago-Iwoye. Were later used for screening of antibiotic Bacillus producers during this dissertation work were shown in table 1. The results of the microbial analysis reveals that the bacterial loads of each soil varied with the count bacteria from the SF-3 (3.20 x 106 cfu/g) while the soil sample from the SF-4 ( 2.20 x 106 cfu/g) has the lowest microbial load.
        Most of other soil sample have Bacillus population of between 2.30 x 106 and 3.10 x 106 cfu/g. the test microorganism were (Bacillus subtilis, Actinomycets spp, licheniformis, Streptomyces spp) were confirmed using basic morphological and biochemical analyses as shown in table 3.
        Description of Table 1: location where the farm soil collected represent school farm 1 to school farm 7, while the sample code was represented by SF-1 to SF-7 respectively.
        
          
             
        
        
        
        
        Table 1: Total Bacillus population count in different farm soil samples from OOU Ago-Iwoye campus
        Sample Location	Sample Code	Dilution factor	Number of colonies	Cfu/g
        School farm 1	Sf 1	10-2	24	2.4 x 106
        School farm 2	Sf 2	10-2	23	2.3 x 106
        School farm 3	Sf 3	10-2	32	3.2 x 106
        School farm 4	Sf 4	10-2	22	2.2 x 106
        School farm 5	Sf 5	10-2	29	2.9 x 106
        School farm 6	Sf 6	10-2	30	3.0 x 106
        School farm 7	Sf 7	10-2	31	3.1 x 106
        
        
        
        
        
        
        
        
        The total Bacillus population counts in different farm soil sample outside OOU environment used for screening of antibiotic Bacillus were shown in table 2, the result of the microbial analysis reveal that the bacteria load of each soil varied with the count bacteria from the OSF-7(5.2 x 106 cfu/g) while the count bacteria from the OSF-3 (4.10 x 106 cfu/g) has the lowest microbial load. Most of the other soil sample also have Bacillus population of between 4.2 x 106 and 5.10 x 106 cfu/g. the test microorganism were (Bacillus subtilis, Actinomycetes spp, Licheniformis, streptomycetes spp) were confirmed using basic morphological and biochemical analysis as also shown in Table 3.
        Description of Table 2: Location where the farm soil collected represent outside the school farm 1 to outside the school farm 7, while the sample code was represented by OSF-1 to OSF-7.
        
        
        
        
        
        
        
        
        Table 2: Total Bacillus population count in different farm soil outside the school environment (OSF)
        Sample Location	Sample Code	Dilution factor	Number of colonies	Cfu/g
        Outside school farm 1	OSF 1	10-2	42	4.2 x 106
        Outside School farm 2	OSF 2	10-2	47	4.7 x 106
        Outside School farm 3	OSF 3	10-2	41	4.1 x 106
        Outside School farm 4	OSF 4	10-2	49	4.9 x 106
        Outside School farm 5	OSF 5	10-2	44	4.4 x 106
        Outside School farm 6	OSF 6	10-2	51	5.1 x 106
        Outside School farm 7	OSF 7	10-2	53	5.3 x 106
        
        
        
        
        
        
        
        The isolates colonies were either circular (5 isolates) or irregular (9 isolates) but had variation in both margin. Moreover, all strains were Bacilli shaped, Gram-positive and mostly single in arrangement.
        All the isolates show the ability of certain bacteria to decompose the ammonia acid. There is no reddish or rose pink color development which means all the isolate indicate a negative result with no color changes.
        In addition, the result shows that the isolate have the ability to produce gas while films which caused by the liberation of free oxygen, 6 shows that the oxidase reaction are positive while the remaining 8 isolate produce negative reaction because of absence of deep purple coloration. The only four sugar were used in the dissertation to test the sugar fermentation of the isolate which were glucose, fructose, lactose and sucrose. Out of these four sugar, only fructose and sucrose has slightly negative reaction while glucose and lactose were all positive.
        From the observation it shows that the microbial load on the farm soil sample outside the school environment is much more than the school farm soil sample, the reason may be the environmental factor of the populace
        Total of 14 Bacillus species were obtained from farm soil sample collected from both OOU Ago-Iwoye campus and Oru community were screened for antimicrobial activity.        
        
        
        
        Table 3: Show the cultural and cell morphology characteristics of all 14 bacillus isolates
        Sample code	Cell morphology	Colony appearance	Gram stain	Indole	Catalose	Oxides	Fructose	Sucrouse	Glusoce	Lactose	Probable organisms
        SF 1	Bacilli	Irregular	+	-	+	+	-	+	+	+	Bacillus subtilis
        SF 2	Bacilli	Irregular	+	-	+	+	+	+	+	+	Bacillus subtilis
        SF 3	Bacilli	Irregular	+	-	+	-	+	+	+	+	Actinomycetes spp
        SF 4	Bacilli	Irregular/transparent	+	-	+	-	+	-	+	+	Bacillus licheniformis
        SF 5	Bacilli	Circular	+	-	+	-	+	+	+	+	Bacillus licheniformis
        SF 6	Bacilli	Circular	+	-	+	-	+	+	+	+	Streptomycetes spp
        SF 7	Bacilli	Irregular	+	-	+	-	+	+	+	+	Actinomycetes spp
        OSF 1	Bacilli	Irregular	+	-	+	-	+	+	+	+	Bacillus licheniformis
        OSF 2	Bacilli	Circular	+	-	+	+	-	+	+	+	Bacillus subtilis
        OSF 3	Bacilli	Circular	+	-	+	+	+	+	+	+	Bacillus subtilis
        OSF 4	Bacilli	Circular	+	-	+	-	+	+	+	+	Streptomycetes spp
        OSF 5	Bacilli	Irregular	+	-	+	+	-	-	+	+	Bacillus subtilis
        OSF 6	Bacilli	Irregular	+	-	+	-	+	+	+	+	Actinomycetes spp
        OSF 7	Bacilli	Circular	+	-	+	+	+	+	+	+	Bacillus subtilis
        
        
         
        Table 4: Show the physical characteristics of all the 14 Bacillus Isolates
        PH	Temperature
        Code	3	7	10	350c	450c
        SF - 1	-	++	++	++	++
        SF -2	+	++	+	++	++
        SF - 3	+	+	++	++	+++
        SF - 4	+	++	++	++	++
        SF - 5	+	+	++	++	++
        SF - 6	+	+	++	++	+++
        SF - 7	+	+	++	++	++
        OSF - 1	+	+	++	++	+
        OSF - 2	+	+	++	++	+++
        OSF - 3	+	++	++	++	++
        OSF - 4	+	++	++	++	+
        OSF - 5	-	+	++	++	+++
        OSF - 6	-	+	++	++	++
        OSF - 7	+	++	++	++	++
        
        For physical tests, the sign “-‘’ indicate no growth, 
        The sign “+’’ indicate that there is slightly growth 
        Sign “++’’ indicate that there is intermediate growth
        Sign “+++” indicate that there is heavy growth
        
        Table 5: Antimicrobial production pattern of different bacillus isolate from soil sample
        Antibiotic Production	Antibiotic production range	Number of Antibiotic producer	Percentage of Antibiotic Production
        Large	1-7mm	6	43%
        Average	7-9mm	3	22%
        Slight	9-12mm	2	14%
        Average	12-15mm	3	22%
            1-15mm	14	100%
         
        CHAPTER FIVE
        5.0 DISCUSSION AND CONCLUSION 
        5.1 DISCUSSION
        Soil samples are commonly evaluated for isolation of the antibiotic producing organism, because soil microorganism produce lots of antibiotic in order to survive in such a competitive environment. Bacteria producing high number of medically and agriculturally important antibiotic belong to genera bacillus, Streptomyces and pseudomonas [Yoshiko et al., 1998; sharga et al., 2004].
        As the 6 isolate from 14 were Bacillus subtilis, and 3 isolate were Bacillus licheniformis, these results correlates with many previous studies that Bacillus spp are notorious producing antibiotics. It has been documented that bacillus genus and other spore forming bacteria carry genus for the breakdown of diverse carbon source and production of antibiotics [Prescott et al., 2008]. Bacillus species are ubiquitous in nature. Many strain of the genus Bacillus have the capability to synthesize a wide range of antibiotic producing bacillus species from soil samples.
        Several hundred Bacillus subtilis strains have been described with the ability to synthesize more than two dozen antibiotics with diverse structure. The potential of Bacillus subtilis to synthesize antibiotic has been perceived from 50 years. The large number of antibiotic produced from Bacillus subtilis might reflect the diversity of natural isolate.
        Other bacillus species like bacillus Armyloliquefeciens [koumoustsi et al., 2004], Bacillus brevis produce few antibiotic compared to Bacillus subtilis. Few antibiotic producing Actinomycetes have been isolated in the present work. The reason why only small number of Actinomycetes has been isolated may be the texture of soil and other predominant environmental condition at the study location. Actinomycetes required a larger time to grow compared to other bacteria at it has been documented by previous studies [Ahmet et al., 2013].
        Actinomycetes can use a variety of organic nutrient but special media are often preferable [Rahman et al., 2000; sultan et al., 2002]. Probably this could be the logic why Actinomycetes grow in very small number on the media (nutrient agar) used.
        Streptomyces are the most studied and well known group of Actinomyces. These are Gram positive, filamentous bacteria which have a great capability to synthesize most important secondary metabolites such as antibiotic, antivirals, and antifungals [Dehnad et al., 2010].
        Out of 14 isolates, 2 were Streptomyces having white wrinkled colonies. Although, various biochemical tests were performed but the strains were not identified to the species level for proper identification of genera and species of Actinomyces as well as morphological and physiological characteristic, different other biochemical parameters such as cell wall chemo type, peptidoglycan type, whole cell sugar patter, phospholipid type should be considered [Dehnad et al., 2010].
        
        5.2 CONCLUSION
        It is concluded that Bacillus subtilis, Bacillus licheniformis, Streptomyces and Actinomycetes isolated from OOU farm soil and its environment have the potential to produce antibiotics.
        The dissertation revealed that, bacteria with the potential to produce antibiotics are present in Oru community. A community outside Olabisi Onabanjo University, Ago-Iwoye environment than the school campus farm soil. This work may also contribute in provision preliminary information on the antibiotic producing microorganism especially in the high of the prevalence of the antibiotic multi resistance bacterial pathogen in health care system.
        Further work should be carried out on the isolates while additional exploratory work could be made by checking other soil around Ago-Iwoye, environment for more novel antibiotic producing bacteria.
        Also,  for antibiotic production and its actively as well as purification and application can be further exploited in the future.</P>
        <p id="bottom">last page</p>
       <a href="#Top">Go to Top</a>
       
       </p>

    
</body>
</html>